pwndbg/exception.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pdb
+import traceback
+
+import pwndbg.config
+import pwndbg.stdio
+
+verbose = pwndbg.config.Parameter('exception-verbose', False, 'whether to print a full stacktracefor exceptions raised in Pwndbg commands')
+debug = pwndbg.config.Parameter('exception-debugger', False, 'whether to debug exceptions raised in Pwndbg commands')
+
+def handle():
+    # Display the error
+    if debug or verbose:
+        print(traceback.format_exc())
+    else:
+        print(e)
```

Comment:
We can probably grab the exception using inspect module ([py2](https://docs.python.org/2/library/inspect.html), [py3](https://docs.python.org/3/library/inspect.html)).